egs/swbd/s5c/RESULTS

```diff
@@ -109,6 +109,20 @@ exit 0
 %WER 11.8 | 1831 21395 | 89.4 7.0 3.6 1.3 11.8 47.8 | exp/nnet3/lstm_ld5_sp/decode_eval2000_sw1_fsh_fg/score_13_0.0/eval2000_hires.ctm.swbd.filt.sys
 %WER 13.2 | 1831 21395 | 88.3 8.0 3.7 1.5 13.2 49.5 | exp/nnet3/lstm_ld5_sp/decode_eval2000_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
 
+# these are results with nnet3 LSTMs cell_dim=1280, recurrent_dim=384, lstm_delay=-1 -2 -3, label_delay=5 num_params=25010228 (8 epoch training on speed-perturbed
+# and volume perturbed data)
+%WER 15.20 [ 7477 / 49204, 1031 ins, 1912 del, 4534 sub ] exp/nnet3/lstm_1280_384_123_ld5_sp/decode_train_dev_sw1_fsh_fg/wer_12_0.0
+%WER 16.29 [ 8013 / 49204, 1121 ins, 1908 del, 4984 sub ] exp/nnet3/lstm_1280_384_123_ld5_sp/decode_train_dev_sw1_tg/wer_11_0.0
+%WER 11.4 | 1831 21395 | 89.8 6.8 3.4 1.2 11.4 46.0 | exp/nnet3/lstm_1280_384_123_ld5_sp/decode_eval2000_sw1_fsh_fg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
+%WER 12.7 | 1831 21395 | 88.6 7.6 3.8 1.3 12.7 48.7 | exp/nnet3/lstm_1280_384_123_ld5_sp/decode_eval2000_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
+
+# these are results with nnet3 bidirectional LSTMs cell_dim=1024, recurrent_dim=128, lstm_delay=[-1,1] [-2,2] [-3,3] num_params=20101172 (8 epoch training on
```

Comment:
I see that the run.sh script does not have the commands for LSTM script. Could you please add a command for that too.